src/domain/networking/LimitedNodeList.ts

```diff
@@ -57,63 +57,64 @@ class LimitedNodeList {
     static INVALID_PORT = -1;
 
 
-    #_nodeSocket;  // Socket
+    protected _nodeSocket: Socket;
+    protected _localSockAddr = new SockAddr();
+    protected _publicSockAddr = new SockAddr();
+    protected _packetReceiver: PacketReceiver;
 
-    #_localSockAddr = new SockAddr();
-    #_publicSockAddr = new SockAddr();
 
-    #_packetReceiver;  // PacketReceiver
-
-
-    constructor(ownerType = NodeType.DomainServer, socketListenPort = LimitedNodeList.INVALID_PORT,  // eslint-disable-line
-        dtlsListenPort = LimitedNodeList.INVALID_PORT, _privateFields = {}) {  // eslint-disable-line
+    // eslint-disable-next-line
```

Comment:
Remove multiple eslint-disable-*.
Maybe add a comment one why these need disabling.

Reply:
This is all to deal to both TypeScript and ESlint complaining about unused parameters. I'll remove the unused parameters until such time as we want to use them.